autograder/api/config.py

```diff
@@ -299,6 +299,19 @@ def __call__(self, parser, namespace, raw_values, option_string = None):
     required = False,
     parser_options = {'action': 'store_true', 'default': False})
 
+PARAM_PROXY_EMAIL = APIParam('proxy-email',
+    'The email of the user that is the target of this proxy request.',
```

Comment:
I think this is a little too obscure for a normal user.
Use some more direct words like "fake" or "pretend".

Maybe something Like:
"The email of the user the pretend to make a submission as."

Or maybe just add a parenthetical at the end, e.g., " (the user the submission is submitted under).".